src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -1,81 +1,152 @@
-        </Card.Header>
-        <Divider />
-        <Card.Body pb={0} px={0}>
-          <Stack spacing={4}>
-            <Row>
-              <HelperTooltip label='This is the rate'>
-                <Row.Label>Rate</Row.Label>
-              </HelperTooltip>
-              <Box textAlign='right'>
-                <Text>{`1 ${sellAsset.currency.symbol} = ${firstNonZeroDecimal(bn(quote.rate))} ${
-                  buyAsset.currency.symbol
-                }`}</Text>
-                <Text color='gray.500'>@{trade?.name}</Text>
-              </Box>
-            </Row>
-            <Row>
-              <HelperTooltip label='This is the Miner Fee'>
-                <Row.Label>Miner Fee</Row.Label>
-              </HelperTooltip>
-              <Row.Value>{toFiat(bn(fees?.fee).times(quote?.rate).toNumber())}</Row.Value>
-            </Row>
-            <Row>
-              <HelperTooltip label='This is the Miner Fee'>
-                <Row.Label>ShapeShift Fee</Row.Label>
-              </HelperTooltip>
-              <Row.Value>$0.00</Row.Value>
-            </Row>
-          </Stack>
-        </Card.Body>
-        <Card.Footer px={0} py={0}>
-          <Button
-            colorScheme='blue'
-            size='lg'
-            width='full'
-            mt={6}
-            onClick={() => history.push('/trade/input')}
-          >
-            Confirm and Trade
-          </Button>
-        </Card.Footer>
-      </Card>
+      <Box as='form' onSubmit={handleSubmit(onSubmit)}>
+        <Card variant='unstyled'>
+          <Card.Header px={0} pt={0}>
+            <SimpleGrid gridTemplateColumns='25px 1fr 25px' alignItems='center' mx={-2}>
+              <IconButton
+                icon={<ArrowBackIcon />}
+                aria-label='Back'
+                variant='ghost'
+                fontSize='xl'
+                isRound
+                onClick={() => history.push('/trade/input')}
+              />
+              <Card.Heading textAlign='center'>
+                <Text translation='trade.confirmTrade' />
+              </Card.Heading>
+            </SimpleGrid>
+            <AssetToAsset buyAsset={buyAsset} sellAsset={sellAsset} mt={6} status={status} />
+          </Card.Header>
+          <Divider />
+          <Card.Body pb={0} px={0}>
+            <Stack spacing={4}>
+              {txid && (
+                <Row>
+                  <Row.Label>
+                    <RawText>Tx ID</RawText>
+                  </Row.Label>
+                  <Box textAlign='right'>
+                    <Link
+                      isExternal
+                      color='blue.500'
+                      href={sellAsset.currency?.explorerTxLink + txid}
+                    >
+                      <Text translation='trade.viewTransaction' />
+                    </Link>
+                  </Box>
+                </Row>
+              )}
+              <Row>
+                <HelperTooltip label='This is the rate'>
```

Comment:
translations

Reply:
This is just a label for the tool tip. Not user facing.

Reply:
tooltips aren't user facing? isn't the tooltip what shows up when they hover over the ? icon?